### PR TITLE
HOTFIX: kafka streams lib missing in dependencies.gradle

### PR DESCRIPTION
**Pull request by ableegoldman**
Seems this was missed to add during the original 2.6.0 release

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -166,6 +166,7 @@ libs += [
   kafkaStreams_23: "org.apache.kafka:kafka-streams:$versions.kafka_23",
   kafkaStreams_24: "org.apache.kafka:kafka-streams:$versions.kafka_24",
   kafkaStreams_25: "org.apache.kafka:kafka-streams:$versions.kafka_25",
+  kafkaStreams_26: "org.apache.kafka:kafka-streams:$versions.kafka_26",
```
Comment (ableegoldman) [inline gradle/dependencies.gradle:169]: I'm not 100% sure if this is actually supposed to be on the 2.6 branch or not, but the `kafka_26` was already defined and it seems odd to have one but not the other. If adding this is wrong, I can take out the `kafka_26` instead 🤷‍♀️ . @mjsax ?

Comment (mjsax) [reply gradle/dependencies.gradle:169]: We don't need this for `2.6` branch, as when we test upgrade from older version to `2.6` we just use the jar we just built base on `2.6` branch -- we only need to pull in older versions.

Comment (ableegoldman) [reply gradle/dependencies.gradle:169]: So I can/should remove the `kafka_26` from this branch as well, yes?

Comment (mjsax) [reply gradle/dependencies.gradle:169]: Well, this PR adds `kafka_26` but we don't need to add it -- not sure what you want to remove?

Comment (ableegoldman) [reply gradle/dependencies.gradle:169]: not ` kafkaStreams_26`, `kafka_26` (defined above) -- it just seems weird to me that we would define one but not the other? But I guess it doesn't hurt so I can close this PR

Comment (mjsax) [reply gradle/dependencies.gradle:169]: Ah. Yes. Could be removed. It's unused. But also does not matter to have an unused var defined...

Comment (mjsax) [reply gradle/dependencies.gradle:169]: Seems this PR added if for no reason (but also no harm): https://github.com/apache/kafka/commit/9ebc71e3d7d1e361b3844d907c4ae935fcda1240

**Pull request closed without merging**